.travis.yml

```diff
@@ -16,14 +17,11 @@ addons:
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0"
```

Comment:
I'm not too sure if it has a real impact, but in the template the line should include ```MAKEPOT="0"```
I guess this would avoid useless .pot generation (possibly we want to generate it only once).
see https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L32